web/src/engine/websites/Tapread.ts

```diff
@@ -0,0 +1,98 @@
+import { Tags } from '../Tags';
+import icon from './Tapread.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchCSS, FetchJSON, FetchRequest } from './../FetchProvider';
+
+const imgURL = 'https://static.tapread.com';
+
+type APIMangas = {
```

Comment:
It seems not all these fields are used in the implementation.
Suggestion to only declare fields that are actually referenced in source code.
Other API type definitions as well...